src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -60,33 +65,41 @@ export class AppointmentsPage extends Component {
       showScheduleButton,
       showCommunityCare,
       hasExpressCareAccess,
+      expressCare,
       showDirectScheduling,
       isCernerOnlyPatient,
       showPastAppointments,
     } = this.props;
+    const { fetchWindowsStatus, allowRequests } = expressCare;
 
     return (
       <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--2p5">
         <Breadcrumbs />
         <div className="vads-l-row">
           <div className="vads-l-col--12 medium-screen:vads-l-col--8 vads-u-margin-bottom--2">
             <h1 className="vads-u-flex--1">{pageTitle}</h1>
-            {showScheduleButton && (
-              <ScheduleNewAppointment
-                isCernerOnlyPatient={isCernerOnlyPatient}
-                showCommunityCare={showCommunityCare}
-                showDirectScheduling={showDirectScheduling}
-                startNewAppointmentFlow={this.startNewAppointmentFlow}
-              />
-            )}
-            {hasExpressCareAccess && (
+            {fetchWindowsStatus === FETCH_STATUS.loading ? (
```

Comment:
Similar to what I said above, I think we need to make sure this doesn't block loading the page when the feature toggle is off.